### DISCUSSION TITLE
GB Blankcheck not empty

### DISCUSSION THREAD

**Discussion by acocalypso** [sanni/cartreader#77] [Coding Corner]
I added some gb flash ids to the code like:

else if (strcmp(flashid, "FF00") == 0) {
      println_Msg(F("M5M5256DRV"));
      print_Msg(F("Banks: "));
      print_Msg(romBanks);
      println_Msg(F("/64"));
      display_Update();
    }

I try to reflash a chinese repro Pokemon card.
Its detected as MBC3+RAM and when I try to Flash MBC3 I get the info that It tries to erase the flash and then do the blankcheck but its not empty and it fails.

Did I miss something?

**Comment by sanni** [comment]
The M5M5256DRV is an SRAM. The flash is the other chip. You have to read its datasheet and then write a custom erase function accordingly. Just adding an ID is not enough.     

If you get stuck while coding you can also search for your repro PCB on this site https://flashcartdb.com/index.php/Main_Page#GameBoy_Clone.2FRepo_Flash_Carts and if found look at its erase function in the GBxCart RW source code https://github.com/insidegadgets/GBxCart-RW      

Ofc buying a GBxCart RW is also an option, it's a very good dumper for GB/A.
